src/platform/unix.rs

```diff
@@ -93,100 +92,42 @@ impl<T> EventLoopWindowTargetExtUnix for EventLoopWindowTarget<T> {
     }
 }
 
-/// Additional methods on `EventLoop` that are specific to Unix.
-pub trait EventLoopExtUnix {
-    /// Builds a new `EventLoop` that is forced to use X11.
-    ///
-    /// # Panics
-    ///
-    /// If called outside the main thread. To initialize an X11 event loop outside
-    /// the main thread, use [`new_x11_any_thread`](#tymethod.new_x11_any_thread).
+/// Additional methods on [`EventLoopBuilder`] that are specific to Unix.
+pub trait EventLoopBuilderExtUnix {
```

Comment:
Could we maybe separate `X11` and `Wayland` here? I'd like to see something like that for Wayland, since it'd allow to unsubscribing from events that winit handles but downstream doesn't care (which will require extra protocols, etc, etc). I'm not sure that it'll be ergonomic with just `ExtUnix`.

Reply:
I agree that we should consider splitting the traits in `platform::unix` into separate traits, but would you be okay with doing that in a separate PR? I can whip one up if you want.

Reply:
I mean not all traits, but precisely this one, so we would have proper API? Though, it's not like it'll change a lot, you'd just end up rewriting changelog entry, since you add one thing and remove it right away. I think doing it properly right away is the way to go, and it shouldn't be that hard to implement.

Reply:
All the trait methods in `platform::unix` (except `MonitorHandleExtUnix::native_id`) are already `cfg`'d to the specific backend, and if we don't intend changing those I would argue we should keep the current design for consistency.

And what about `EventLoopBuilderExtUnix::any_thread`? Should we keep `EventLoopBuilderExtUnix` for that, or should it now be present in both traits?

Reply:
I've rethought things a bit and I'd suggest to go with the current `Unix` approach, since we'd need this trait anyway to have code composition seamlessly in mixed x11/wayland environments.

So let's keep it for later design.